docs/security/cis_self_assessment16.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: CIS Self-Assessment Guide
+title: CIS v1.6 Self-Assessment Guide
```

Comment:
Can you make the titles just "CIS 1.5 Self-Assessment" and "CIS 1.6 Self-Assessment" to avoid unnecessary text wrapping in the sidebar?

![image](https://user-images.githubusercontent.com/370103/119883771-c3b25f80-bee4-11eb-9fc0-d30495e78fe5.png)

Reply:
Done :)